enrolments/tests/test_serializers.py

```diff
@@ -235,32 +218,110 @@ def test_enrolment_serializer(self):
                     "year": self.session.year,
                 },
                 "preferred_class": {
-                    "id": self.class1.id,
-                    "name": self.class1.name,
+                    "id": self.preferred_class.id,
+                    "name": self.preferred_class.name,
                 },
                 "enrolled_class": {
-                    "id": self.class2.id,
-                    "name": self.class2.name,
+                    "id": self.enrolled_class.id,
+                    "name": self.enrolled_class.name,
                 },
                 "status": self.enrolment.status,
             },
             EnrolmentSerializer(self.enrolment).data,
         )
 
-    def test_enrolment_update(self):
-        serializer = EnrolmentSerializer(self.enrolment, data=self.update_request)
-        self.assertTrue(serializer.is_valid())
 
-    def test_enrolment_update__read_only_session(self):
-        data = dict(self.update_request)
-        data["session"] = self.other_session.id
-        serializer = EnrolmentSerializer(self.enrolment, data=data)
+class EnrolmentCreateSerializerTestCase(TestCase):
```

Comment:
this file is getting a bit long, so let's start breaking tests out into their own files - could we rename this file to `enrolments/tests/serializers/test_enrolment_create.py`?